aamt_deploysite.pl

```diff
@@ -0,0 +1,159 @@
+use strict;
+# package used to get the current working folder
+use aamt_wrkFolder;
+my $currentLibfolder;
+# get the current working folder and unset the environment variable that has been set in the 
+# batch file/ shell script file 
+BEGIN {$currentLibfolder = main_getCurrentLibFolder();}
+use vars qw/$currentLibfolder/;
+use lib $currentLibfolder;
+
+#Standard and nonstandard packages used by the tool 
+use strict;                 #for declaration of variables prior to use
+use FileHandle;             #for file operations
+use File::Copy;             #for copying the file in a local machine  
+use File::Path;
+use LWP::Simple;            #for website header information
+use IO::Dir;			    #for directory handling
+use Sys::Hostname;          # 
+use File::Listing;          #to display a list of files in a directory  
+use Time::localtime;        #get system time information
+
+# custom packages developed to be used by the tool	
+use aamt_constants;		    #for user input or error or file constants 
+use aamt_utilityFunctions;  #for common procedures
+use aamt_informationLog;    #used for logging information into Log,Status or Recovery file
+use aamt_userInterface;     #used for user interface definition
+use aamt_auth;	            #used for Authentication and user query module 	
+use aamt_parse;             #used for parsing the conf file and getting the site information	
+use aamt_parse2;		    #used for parsing the conf file and generating the 2D array.
+
+# main subroutine starts here
```

Comment:
Plan to merge the aamt_deploysite.pl,aamt_listsites.pl and the original migrate_tool_main.pl into one main script like aamt.pl, it works as the below:
1. Display usage help when run
sudo perl aamt.pl 
or
sudo perl aamt.pl -help
2. Keep the original interactive behavior when run
sudo perl aamt.pl -i
or
perl aamt.pl -I
3. Run in non-interactive when run
sudo perl aamt.pl list test01 /etc/apache2/apache2.conf   
or
sudo perl aamt.pl deploy test01 /etc/apache2/apache2.conf 2 /home/azureadmin/testmig102.chinacloudsites.cn.PublishSettings
Will got ahead to implement after get your comments or suggestions.